CHANGELOG.md

```diff
@@ -94,3 +94,17 @@ release.
   ([#17](https://github.com/open-telemetry/opentelemetry-specification/pull/17))
 - Mark service.version as stable.
   ([#106](https://github.com/open-telemetry/semantic-conventions/pull/106))
+- BREAKING: Add namespaces to JVM metric attributes ([#20](https://github.com/open-telemetry/semantic-conventions/pull/20)).
+  - Memory metric attribute renames
+    - Applies to metrics
+      - `metric.process.runtime.jvm.memory.usage`
```

Comment:
Hi @zeitlinger is this PR still a WIP? Because the metrics in here are not the ones actually changed.

Reply:
I think I copy-pasted from the wrong lines - should be fixed now.